### PR TITLE
Documentation for /etc/leofs configs and multiple node instances

### PR DESCRIPTION
**Pull request by vstax**
Documentation for #791 and #792

### CODE REVIEW

**Comment by vstax** [comment]
"/usr/local/leofs/current/leo_manager_1/bin/leo_manager_1 start" needs to be replaced with "/usr/local/leofs/current/leo_manager_1/bin/leo_manager start"

**Comment by mocchira** [comment]
@vstax WIP. anyway from the next PR to docs, please follow the below instructions
```bash
### at the top dir on leofs
mkdocs gh-deploy
```
before PR then this will generate the site on your git hosting place like (ex. https://mocchira.github.io/leofs) and allow us to review docs easily.

File: docs/admin/system_admin/multiple_nodes.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+obj_containers.path
+leo_ordning_reda.temp_stacked_dir
+```
+For gateways running in HTTP mode (S3 / REST), additionally:
+```
+http.port
+http.ssl_port
+```
+
+Please note that this list might be not complete as new options might be added to config files. When using this feature, **it's up to user to ensure that all paths above point to different directories**. Trying to share the same ports or snmp_conf setting between nodes would result in second instance failing to start, but trying to share log or queue directory could lead to disasterous results. **Always double-check that all paths are different before using this feature.**
```
Comment (mocchira) [inline docs/admin/system_admin/multiple_nodes.md]: s/disasterous/disastrous/

**Comment by vstax** [comment]
@mocchira Thank you for "mkdocs gh-deploy" advice, I had no idea docs deployment is so automated.
Anyhow, I fixed these typos and published docs to https://vstax.github.io/leofs/

After merging this, I think it should be deployed to main site as well, so I can give proper documentation references in email about upcoming changes.

**Comment by mocchira** [comment]
Thanks for your contribution.

**Comment by yosukehara** [comment]
@vstax Thank you for your contribution. I've published them to our website:

* [For Administrators  System Administration  Multiple Node Instances](https://leo-project.net/leofs/docs/admin/system_admin/multiple_nodes/)
* [For Administrators  System Administration  Persistent Configuration](https://leo-project.net/leofs/docs/admin/system_admin/persistent_configuration/)
* [For Administrators  System Administration  System Migration](https://leo-project.net/leofs/docs/admin/system_admin/migration/)

**Pull request merged**